apps/prairielearn/src/components/SyncProblemButton.html.tsx

```diff
@@ -1,30 +1,39 @@
-import { escapeHtml, html, unsafeHtml } from '@prairielearn/html';
-
 import { ansiToHtml } from '../lib/chalk.js';
+import { renderHtml } from '../lib/preact-html.js';
 
-export function SyncProblemButton({ output, type }: { output: string; type: 'error' | 'warning' }) {
+export function SyncProblemButtonJsx({
+  output,
+  type,
+}: {
+  output: string;
+  type: 'error' | 'warning';
+}) {
   const title = type === 'error' ? 'Sync Errors' : 'Sync Warnings';
   const classes =
     type === 'error' ? 'fa-times text-danger' : 'fa-exclamation-triangle text-warning';
 
-  const popoverContent = html`<pre
-    class="text-white rounded p-3 mb-0"
-    style="background-color: black;"
-  >
-${unsafeHtml(ansiToHtml(output))}</pre
-  >`;
+  const popoverContent = renderHtml(
+    <pre class="text-white rounded p-3 mb-0" style="background-color: black;">
+      {ansiToHtml(output)}
+    </pre>,
+  );
 
-  return html`
+  return (
     <button
       class="btn btn-xs btn-ghost me-1"
+      type="button"
       data-bs-toggle="popover"
       data-bs-container="body"
       data-bs-html="true"
-      data-bs-title="${title}"
-      data-bs-content="${escapeHtml(popoverContent)}"
+      data-bs-title={title}
+      data-bs-content={popoverContent}
```

Comment:
👀 just want to confirm that this is tested. This is _probably_ correct but getting the escaping correct on these attributes has bitten us before so I wanted to check.

Reply:
<img width="938" height="438" alt="Image" src="https://github.com/user-attachments/assets/168e69d9-b595-4448-b19f-313b87069142" />

I had to turn off `escapeHtml` for a non-mangled output here. Done in 645b022b647c5a6f5dae1472e946d038eee4e8a8.